osbs/core.py

```diff
@@ -866,6 +866,24 @@ def restore_resource(self, resource_type, resource):
         check_response(response)
         return response
 
+    def create_config_map(self, config_name, config_data):
+        url = self._build_k8s_url("configmaps/")
+        response = self._post(url, data=json.dumps(config_data))
```

Comment:
I think we need more than this. `config_data` will be something like `{"metadata.json": "{...}"}`, i.e. the `data` part of the v1.ConfigMap: https://docs.openshift.org/latest/rest_api/kubernetes_v1.html#create-a-configmap

See `create_image_stream` for the equivalent for creating a v1.ImageStream: it reads a template, image_stream.json, and adjusts fields appropriately. I think we probably want to do the same sort of thing as that.